From 8dd6ff8d6a0a3173bded683ecf85834cce7cf297 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 9 Aug 2016 13:01:32 +0200
Subject: [PATCH] fips: remove sha1-mb

This module should be loaded automatically, if the HW supports it.
---
 modules.d/01fips/module-setup.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/modules.d/01fips/module-setup.sh b/modules.d/01fips/module-setup.sh
index 43d170f1..dc476347 100755
--- a/modules.d/01fips/module-setup.sh
+++ b/modules.d/01fips/module-setup.sh
@@ -22,7 +22,6 @@ installkernel() {
         _fipsmodules+="rmd160 rmd256 rmd320 rot13 salsa20 seed seqiv serpent sha1 sha224 sha256 sha256_generic "
         _fipsmodules+="aes_s390 des_s390 prng sha256_s390 sha_common des_check_key sha1_s390 sha512_s390 "
         _fipsmodules+="sha384 sha512 sha512_generic tcrypt tea tnepres twofish wp256 wp384 wp512 xeta xtea xts zlib "
-        _fipsmodules+="sha1-mb"
     fi
 
     mkdir -m 0755 -p "${initdir}/etc/modprobe.d"
